.github/workflows/docs.yml

```diff
@@ -1,56 +1,57 @@
-#
-# Copyright (c) 2022 alandefreitas (alandefreitas@gmail.com)
```

Comment:
Why remove the copyright?

Reply:
Omg, I didn't see that, sorry! I'll put that back.